docs/source/contributor-guide/howtos.md

```diff
@@ -19,6 +19,12 @@
 
 # HOWTOs
 
+## How to update the version of Rust used in CI tests
+
+- Make a PR to update the [rust-toolchain] file in the root of the repository:
+
+[rust-toolchain]: https://github.com/apache/datafusion/blob/main/rust-toolchain.toml
```

Comment:
Does this apply to CI only, or is rust-toolchain read by cargo too?

Reply:
I think it applies by default if you just run `cargo` locally if that is what you are asking

So unless someone overrides the setting by default local development environments would use the toolchain version specified by this file